doc/reference/devices_nic.md

```diff
@@ -1,410 +1,454 @@
-
-These are detailed in the "Managed" column in the NIC specific sections below.
+Most NICs support hotplugging for both containers and VMs.
+See the description for each NIC type for exceptions.
+```
 
-## NICs Available
+Network devices, also referred to as *Network Interface Controllers* or *NICs*, supply a connection to a network.
+LXD supports several different types of network devices (*NIC types*).
 
-See the settings for the NIC below for details about which properties are available.
+## `nictype` vs. `network`
 
-The following NICs can be specified using the `nictype` or `network` properties:
+When adding a network device to an instance, there are two methods to specify the type of device that you want to add: through the `nictype` device option or the `network` device option.
 
-- [`bridged`](#nic-bridged): Uses an existing bridge on the host and creates a virtual device pair to connect the host bridge to the instance.
-- [`macvlan`](#nic-macvlan): Sets up a new network device based on an existing one but using a different MAC address.
-- [`sriov`](#nic-sriov): Passes a virtual function of an SR-IOV enabled physical network device into the instance.
+These two device options are mutually exclusive, and you can specify only one of them when you create a device.
+However, note that when you specify the `network` option, the `nictype` option is set automatically.
 
-The following NICs can be specified using only the `network` property:
+`nictype`
+: When using the `nictype` device option, you specify a network interface that is not controlled by LXD.
+  Therefore, you must specify all information that LXD needs to use the network interface.
 
-- [`ovn`](#nic-ovn): Uses an existing OVN network and creates a virtual device pair to connect the instance to it.
+  When using this method, the `nictype` option must be specified when creating the device, and it cannot be changed later.
 
-The following NICs can be specified using only the `nictype` property:
+`network`
+: When using the `network` device option, the NIC is linked to an existing {ref}`managed network <managed-networks>`.
+  In this case, LXD has all required information about the network, and you need to specify only the network name when adding the device.
 
-- [`physical`](#nic-physical): Straight physical device pass-through from the host. The targeted device will vanish from the host and appear in the instance.
-- [`ipvlan`](#nic-ipvlan): Sets up a new network device based on an existing one using the same MAC address but a different IP.
-- [`p2p`](#nic-p2p): Creates a virtual device pair, putting one side in the instance and leaving the other side on the host.
-- [`routed`](#nic-routed): Creates a virtual device pair to connect the host to the instance and sets up static routes and proxy ARP/NDP entries to allow the instance to join the network of a designated parent interface.
+  When using this method, LXD detects and sets the `nictype` option automatically.
```

Comment:
Same as https://github.com/lxc/lxd/pull/11183/commits/29d72e565686908e368a46383be3e8ff2050ab04#r1037234987 perhaps "derived" would be more accurate here.